packages/vscode-extension/src/node/metafieldDefinitions.ts

```diff
@@ -0,0 +1,36 @@
+import * as child_process from 'child_process';
+import { promisify } from 'node:util';
+
+const exec = promisify(child_process.exec);
+
+const isWin = process.platform === 'win32';
+
+export async function fetchMetafieldDefinitions() {
+  const path = getShopifyCliPath();
```

Comment:
The current CLI does not have the ability to pull metafields, so tophat this build the CLI PR [here](https://github.com/Shopify/cli/pull/4782) and hardcode this value to the path.

E.g.

`const path = '<where-ever-you-cloned-it>/cli/packages/cli/bin/dev.js'`